server/ansible/roles/pbench-server-activate-httpd-setup-restart/tasks/main.yml

```diff
@@ -25,9 +25,9 @@
     name: rsync
     state: latest
 
-- name: "Copy the web-server bits to the <pbench>/public_html/static directory."
-  synchronize:
-    src: "{{ pbench_server_install_dir }}/html/static/"
-    dest: "{{ pbench_dir }}/public_html/static"
-    recursive: yes
-  delegate_to: "{{ inventory_hostname }}"
+#- name: "Copy the web-server bits to the <pbench>/public_html/static directory."
+#  synchronize:
+#    src: "{{ pbench_server_install_dir }}/html/static/"
+#    dest: "{{ pbench_dir }}/public_html/static"
+#    recursive: yes
+#  delegate_to: "{{ inventory_hostname }}"
```

Comment:
I believe Pete did that part manually for this deployment, but wanted to fix it before submit; this was getting rid of a messy dependency to make the synchronize work that apparently proved problematic on RHEL 8. (If I'm correctly capturing Pete's explanation yesterday.)

This _is_ still a draft PR, so it's in no immediate danger of merging.

Reply:
Yup, yup.  I was just putting in a plea for some explanation.  You and I understand the needs and forces at work.  Adding a line or two explaining why the code wasn't outright deleted and/or the criteria which should be met in order for the process to be completed (or undone) would ease the minds of reviewers and future developers, and it doesn't take much more time than adding all those `#`'s.

Reply:
I filed this change as a separate patch from the first so that I could just drop it when, or if, we fixed this problem.  But I have not invested any time to find a replacement for this under the `become_user` method.

I'll drop it from this PR, and then just fix it by hand when re-deploying.